lib/Map/prettifyCoordinates.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+import defined from 'terriajs-cesium/Source/Core/defined';
+import defaultValue from 'terriajs-cesium/Source/Core/defaultValue';
+
+/**
+ * Turns the longitude / latitude in degrees into a human readable pretty strings.
+ *
+ * @param {Number} longitude The longitude to format.
+ * @param {Number} latitude The latitude to format.
+ * @param {Object} options Object with the following properties:
+ * @param {Number} options.height The height.
+ * @param {Number} options.errorBar The error +/- for the height.
+ * @param {Number} options.digits The number of digits to fix the lat / lon to.
+ */
+function prettifyCoordinates(longitude, latitude, options) {
+    var result = {};
+
+    options = defaultValue(options, {});
+    options.digits = defaultValue(options.digits, 3);
```

Comment:
Better to assign this to a local variable instead of putting it in `options.digits`.  It's impolite to change an object passed in by the user without a very good reason.

Reply:
Thanks for this, absolutely right...wish we had const correctness so that we could get the complier to detect this ;) Have fixed this for both options and options.digits...don't love the variable name I've used but out of better inspiration atm.

Reply:
I'm not too fussed about `options` itself, since assigning a new value to that won't be visible outside the function. I know lots of people are of the mind that you shouldn't modify parameters, though, and I'm ok with that.